experimental/GaloisGrp/GaloisGrp.jl

```diff
@@ -1244,9 +1244,15 @@ function starting_group(GC::GaloisCtx, K::AnticNumberField; useSubfields::Bool =
     #code from Max...
 
     #TODO: wrap this properly
-    tmp = [GAP.Globals.ConStabilize(GAP.Obj(sort(o), recursive=true), GAP.Globals.OnSetsSets) for o in O]
-    G = Oscar._as_subgroup(G, GAP.Globals.Solve(GAP.Obj(vcat(GAP.Globals.ConInGroup(G.X), tmp))))[1]
-    #@show G = intersect([stabilizer(G, sort(o), on_sets_sets)[1] for o=O]...)[1]
+    if false && hasproperty(GAP.Globals, :ConStabilize)
```

Comment:
if false? This will turn it always off.
Good thinking to leave the old code in the show, wasn't it?

Reply:
I wanted to test that the alternate code really passes the test suite. But I should've marked the PR as draft :-/